src/java.base/share/classes/java/util/jar/JarEntry.java

```diff
@@ -115,7 +115,8 @@ public Attributes getAttributes() throws IOException {
      * to trust the entry signed by the signers.
      *
      * @return the {@code Certificate} objects for this entry, or
-     * {@code null} if none.
+     * {@code null} if none. If non-null, this method returns a new
```

Comment:
I think you can remove the "this method" words as it is implied by being in the "@return".